covid19-etl/tests/test_jhu_to_s3.py

```diff
@@ -167,22 +165,12 @@ def test_jhu_to_s3_illinois(clean_up):
         with open(os.path.join(INPUT_DATA_DIR, filename), "r") as f:
             reader = csv.DictReader(f, delimiter=",", quotechar='"')
             for row in reader:
-                country = None
-                state = None
-                for key in ["Country/Region", "Country_Region"]:
-                    if row.get(key):
-                        country = row[key]
-                        break
-                for key in ["Province/State", "Province_State"]:
-                    if row.get(key):
-                        state = row[key]
-                        break
+                country = row.get("Country/Region", row.get("Country_Region"))
+                state = row.get("Province/State", row.get("Province_State"))
```

Comment:
This looks Wayy better than what I was thinking! 💯 🥇

Reply:
What would be the value of `country` when both the fields are missing? `None`?

Reply:
yes